_articles/host.md

```diff
@@ -35,6 +35,8 @@ Swag will include the following:
 
 For this event you can request up to 30 USD. Please note that you can only apply for the budget after the event happend. To do so, fill out the [request budget](https://goo.gl/forms/fkwubidAAnvK40vF3) form and provide the link to your receipts. Budget requests without a link to receipts will not be considered.
 
+Note: We know that in some cases 30 USD might not be enough. So in cases you've spent more do send us the receipts. We will evaluate them and reimburse you if the expenses are logical in your local.
```

Comment:
I guess it would make sense to mark the "Note:" bold, as we do with the funding note. You can do that by putting two * around it.

```
**Note:** ....
```

Reply:
Also, I would remove the "so": "enough. In cases you've spent more.."

Reply:
Additionally, I'm not sure if "are logical in your local" is gramatically and stylisticly correct. Maybe something around  "appropriate" or "justified" would be more understandable and more expressive?